### DISCUSSION TITLE
Can't install the dependencies

### DISCUSSION THREAD

**Discussion by cebri-ctrl** [tin2tin/Pallaidium#14] [Q&A] [answered]
Hi,

I'm currently stuck at the Install Dependencies step.

Many times I have this message :
WARNING: Ignoring invalid distribution -umpy (c:\program files\blender foundation\blender 3.5\3.5\python\lib\site-packages)

Or this one :
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
torchaudio 2.0.2+cu118 requires torch==2.0.1, but you have torch 2.0.0 which is incompatible.
torchvision 0.15.2+cu118 requires torch==2.0.1, but you have torch 2.0.0 which is incompatible.

So when I try to generate I have "Dependencies needs to be installed in the add-on preferences."

Thanks a lot for your help

**Comment by tin2tin** [comment, marked as answer]
On the first issue, you can go to: (c:\program files\blender foundation\blender 3.5\3.5\python\lib\site-packages and delete the folder called -umpy. It's an old version of numpy pip keeps like this, but it is not needed anymore.

On the other problem, the solution could be similar, however in the console messages you can see another path to side-packages. You should open this folder and delete the folder called torch(with Blender closed), and then open Blender again and try Install All Dependencies again. 

There is a lot of hit and miss when using pip and dependancy installation, and then the modules gets updated and something breaks again and again.

Let me know if this works out for you.
